app/assets/config/decidim_problems_manifest.js

```diff
@@ -0,0 +1,3 @@
+//= link decidim/challenges/add_challenge.js
+//= link decidim/challenges/admin/challenges_form.js
+//= link_directory ../stylesheets/decidim/problems .css
```

Comment:
Same here, be careful, there's a white space between the word "challenges" and ".css"
Furthermore, are *challenge* .js related files are needed here ?